### PR TITLE
Combined release notes for Core Modules and Libs

### PR DESCRIPTION
**Pull request by artemkovalyov**
## What Has Changed?

As part of customer communications, we require to have comprehensive release notes in one place which is our documentation. The changelog can continue to be published in GH as before.
## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [x] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [x] You verified all new and changed links still work (changing the `id` or name of a file can break links)
- [x] You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.

### CODE REVIEW

File: docs/js/release-notes.mdx [hunk 1]
```diff
@@ -28,22 +28,37 @@ It will help you:
...
+To search for supported services use our API Business Hub integration.
 
 :::
 
-## Pregenerated Type-Safe Client Libraries (VDM)
+<!-- Copy and fill
+## 1.xx.x [Core Modules] - January 1, 2021
 
-:::tip What are client libraries?
+[Maven Central](https://search.maven.org/search?q=g:com.sap.cloud.sdk*%20AND%20v:3.34.1) | [Javadoc](https://help.sap.com/doc/55bb1642ea614bf582d514ed34401136/1.0/en-US/index.html)
```
Comment (marikaner) [inline docs/js/release-notes.mdx:43]: Maven? 🧐

Comment (artemkovalyov) [reply docs/js/release-notes.mdx:43]:
```suggestion
[GitHub](https://github.com/SAP/cloud-sdk-js/releases/tag/v1.52.0) | [npm](https://www.npmjs.com/search?q=%40sap-cloud-sdk)
```

Comment (artemkovalyov) [reply docs/js/release-notes.mdx]: Good catch. The template is from Java. Fixed.

File: docs/js/release-notes.mdx [hunk 2]
```diff
@@ -53,24 +68,29 @@ These libraries are not Open Source, but free and distributed under the SAP Deve
...
-### New Functionality
+[GitHub](https://github.com/SAP/cloud-sdk-js/releases/tag/v1.52.0) | [npm](https://www.npmjs.com/search?q=%40sap-cloud-sdk)
 
-- **We released a regular update for pregenerated client libraries (also known as VDM) for the [latest RTC release 2111 of SAP S/4HANA Cloud](https://news.sap.com/2021/10/sap-s4hana-cloud-2111-innovating-business-futures/).**
-  - The update covers all the changes to existing OData services and introduces new ones.
-- **We released a regular update for pregenerated client libraries (also known as VDM) for the latest RTC release 2021 FPS0 of SAP S/4HANA On-premise.**
-  - The update covers all the changes to existing OData services and introduces new ones.
+### Compatibility Notes
+
+- We changed default cache isolation strategy from `IsolationStrategy.Tenant` to `IsolationStrategy.Tenant_User`. This applies when you set `useCache` to `true` while using a `getDestination` method for a destination lookup.
```
Comment (florian-richter) [inline docs/js/release-notes.mdx:77]:
```suggestion
- We changed default cache isolation strategy from `IsolationStrategy.Tenant` to `IsolationStrategy.Tenant_User`. 
  This applies when you set `useCache` to `true` while using a `getDestination` method for a destination lookup.
```

File: docs/js/release-notes.mdx [hunk 3]
```diff
@@ -53,24 +68,29 @@ These libraries are not Open Source, but free and distributed under the SAP Deve
...
+
+### Fixed Issues
 
-## Version 1.27.0 - Aug 20, 2021
+- We now cache a destination only when a `JWT` contains all necessary information about a `Tenant` and a `User`. For example, when using `IsolationStrategy.Tenant_User`, the JWT has to contain both **tenant id** and **user id**.
```
Comment (florian-richter) [inline docs/js/release-notes.mdx:81]:
```suggestion
- We now cache a destination only when a `JWT` contains all necessary information about a `Tenant` and a `User`. 
  For example, when using `IsolationStrategy.Tenant_User`, the JWT has to contain both **tenant id** and **user id**.
```

File: docs/js/release-notes.mdx [hunk 4]
```diff
@@ -53,24 +68,29 @@ These libraries are not Open Source, but free and distributed under the SAP Deve
...
+- If `JWT` has an expiration time we'll use it to override the default cache expiration time of 5 minutes.
```
Comment (florian-richter) [inline docs/js/release-notes.mdx:82]:
```suggestion
- If `JWT` has an expiration time, we'll use it to override the default cache expiration time of 5 minutes.
```

File: docs/js/release-notes.mdx [hunk 5]
```diff
@@ -28,22 +28,37 @@ It will help you:
...
 - Protect yourself from bugs and breaking changes in the future
 
-:::note _Release notes_ for the Open Source components
+:::tip What are pregenerated typed client libraries (VDM)?
 
-For the Open Source modules of the SAP Cloud SDK for JavaScript we maintain [release notes](https://github.com/SAP/cloud-sdk-js/releases) on GitHub.
+For your convenience we [pregenerate client libraries](https://www.npmjs.com/search?q=%40sap%2Fcloud-sdk-vdm-*) (also known as Virtual Data Model or VDM for short) for allow-listed OData services of SAP S/4HANA Cloud, Marketing Cloud, and SAP S/4HANA On-premise.
+These libraries are not Open Source, but free and distributed under the SAP Developer license through [npmjs.com](https://www.npmjs.com/search?q=%40sap%2Fcloud-sdk-vdm-*).
+You can always generate a typed client library yourself by using our [OData](features/odata/generate-odata-client.mdx) and [Open API](features/openapi/generate-openapi-client.mdx) code generators.
+To search for supported services use our [API Business Hub integration](https://blogs.sap.com/2021/09/17/the-sap-cloud-sdk-integrates-with-the-sap-api-business-hub/).
```
Comment (FrankEssenberger) [inline docs/js/release-notes.mdx:36]: the term support services is not clear to me.

Comment (artemkovalyov) [reply docs/js/release-notes.mdx:36]:
```suggestion
To easily search for services and get typed client library for them,  use our [API Business Hub integration](https://blogs.sap.com/2021/09/17/the-sap-cloud-sdk-integrates-with-the-sap-api-business-hub/).
```

Comment (artemkovalyov) [reply docs/js/release-notes.mdx]: Good point, I tried to re-phrase it. We can give it another look later to make the heading of the page more structured.

**Pull request merged**